src/components/pages/ProfileSelect/ProfileSelectContainer.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect, useMemo } from 'react';
+import { useOktaAuth } from '@okta/okta-react';
+
+import { connect } from 'react-redux';
+import { child, parent } from '../../../state/actions';
+
+import RenderProfileSelect from './RenderProfileSelect';
+
+function ProfileSelectContainer({ LoadingComponent, ...props }) {
+  // const { authState, authService } = useOktaAuth();
+  // augment "oktaAuth" to behave like "authService"
+  const { authState, oktaAuth } = useOktaAuth();
+  oktaAuth.getUser = oktaAuth.token.getUserInfo;
```

Comment:
Do we want to create a trello card to move this code block into a wrapper function, or somewhere that will do it once per request and provide in state??

Reply:
I'm sure it can be cleaned up and re-examined. 
One thing I do remember running into...
- For security reasons, useOktaAuth() cannot be called from an external hook. We have to call this from inside of Okta's circle of trust. This works out to be internal to each individual component; No shortcuts.

(This may have changed since 4.x)

Reply:
Oh, to be more clear, I meant an or to be between 'wrapper' and 'function'. :| That reads very specific.

We can pass the oktaAuth object to a function to set the additional functions.

I didn't know about the context that we have to stay within. That kinda stinks. That explains why Ryan added the memorization bits.